Refresher/UI/PipelineForm.cs

```diff
@@ -63,19 +69,31 @@ namespace Refresher.UI;
 
     private void UpdateFormState()
     {
+        // adjust progress bars
         this._progressBar.Value = (int)((this._pipeline?.Progress ?? 0) * 100);
         this._currentProgressBar.Value = (int)(this._pipeline?.CurrentProgress * 100 ?? 0);
         this._progressBar.ToolTip = this._pipeline?.State.ToString() ?? "Uninitialized";
 
         bool enableControls = this._pipeline?.State != PipelineState.Running;
+        
+        // highlight progress bars while patching
         this._currentProgressBar.Enabled = !enableControls;
         this._progressBar.Enabled = !enableControls;
+        
+        // disable other things
+        this._formLayout.Enabled = enableControls;
         this._autoDiscoverButton.Enabled = enableControls && this._pipeline?.AutoDiscover == null;
 
+        // set text of autodiscover button
+        if (this._autoDiscoverCts != null)
+            this._autoDiscoverButton.Text = "Running AutoDiscover... (click to cancel)";
+        else if (this._pipeline?.AutoDiscover == null)
+            this._autoDiscoverButton.Text = "AutoDiscover";
+
         this._button.Text = this._pipeline?.State switch
         {
-            PipelineState.NotStarted => "Execute",
-            PipelineState.Running => "Cancel",
+            PipelineState.NotStarted => "Patch!",
+            PipelineState.Running => "Patching... (click to cancel)",
             PipelineState.Finished => "Complete!",
             PipelineState.Cancelled => "Execute",
```

Comment:
`Cancelled` also needs to be changed to `Patch!` (or maybe, `Cancelled. (click to restart)`